dist/src/main/java/io/camunda/commons/configuration/BrokerBasedConfiguration.java

```diff
@@ -78,6 +86,16 @@ public RestGatewayDisabled disableRestGateway() {
     return new RestGatewayDisabled();
   }
 
+  @ConditionalOnRestGatewayEnabled
+  @Bean
+  public CompositeFilter restApiCompositeFilter() {
+    final List<FilterCfg> filterCfgs = properties.getGateway().getFilters();
+    final List<Filter> filters =
+        new FilterRepository().load(filterCfgs).instantiate().collect(Collectors.toList());
```

Comment:
🙃 Tiny thing, and I haven't tried it, but I think you can just write `new FilterRepository().load(filterCfgs).instantiate().toList()` these days 🤷

Reply:
Good point, I need to change my habit. Thanks!